inc/class-thumbnails.php

```diff
@@ -225,6 +244,18 @@ public function get_from_provider( $return, $data, $url ) {
 				$this->set_youtube_thumbnail( $id, $url );
 			}
 		}
+		else if ( strpos( $url, 'slideshare.net' ) !== false ) {
```

Comment:
Please resort to begin with slideshare in the if statements (should always be in alphabetical ascending order)

Reply:
Done